app/controllers/application_controller.rb

```diff
@@ -5,9 +5,11 @@ class ApplicationController < ActionController::Base
   include TradeTariffFrontend::ViewContext::Controller
   include ApplicationHelper
 
+  before_action :set_cache
```

Comment:
I'm alphabetically sorting these setters. Order won't matter for the Cache-Control header specified in the request I don't think